os/drivers/audio/alc5658.c

```diff
@@ -422,6 +481,11 @@ static int alc5658_getcaps(FAR struct audio_lowerhalf_s *dev, int type, FAR stru
 	DEBUGASSERT(caps && caps->ac_len >= sizeof(struct audio_caps_s));
 	audvdbg("type=%d ac_type=%d\n", type, caps->ac_type);
 
+#ifndef CONFIG_DEBUG
+	if(!caps)
```

Comment:
kindly use flower brackets for all "single statement if check" instances as well